lib/cocoapods/generator/app_target_helper.rb

```diff
@@ -99,6 +99,7 @@ def self.add_xctest_search_paths(target)
       # @return [void]
       #
       def self.add_swift_version(target, swift_version)
+        raise 'Cannot set empty Swift version to target.' if swift_version.nil?
```

Comment:
`blank?` might be what you are looking for here?

Reply:
sure